gateway-service/src/main/java/org/zowe/apiml/gateway/config/DiscoveryClientConfig.java

```diff
@@ -67,7 +67,7 @@ public class DiscoveryClientConfig {
     @Bean
     public List<AdditionalRegistration> additionalRegistration(StandardEnvironment environment) {
         List<AdditionalRegistration> additionalRegistrations = new AdditionalRegistrationParser().extractAdditionalRegistrations(System.getenv());
-        log.debug("Parsed {} additional regs, \t first: {}", additionalRegistrations.size(), additionalRegistrations.stream().findFirst().orElse(null));
+        log.debug("Parsed {} additional registration: {}", additionalRegistrations.size(), additionalRegistrations.toArray(new AdditionalRegistration[0]));
```

Comment:
```suggestion
        log.debug("Parsed {} additional registration: {}", additionalRegistrations.size(), additionalRegistrations);
```
The method `toString` generates a random value, I am not sure about the logger, but anyway I guess it is not necessary to transform.